CONTRIBUTING.md

```diff
@@ -119,11 +119,13 @@ in order to craft an excellent pull request:
    [interactive rebase](https://help.github.com/articles/about-git-rebase/)
    feature to tidy up your commits before making them public.
 
-5. Make sure all the tests are still passing.
+5. Ensure the code is formatted and the sample still works.
 
    ```bash
    dotnet tool restore
-   dotnet fake build
+   dotnet fantomas .
+   dotnet build -c Release
+   dotnet run --project src/FSharp.Analyzers.Cli/FSharp.Analyzers.Cli.fsproj -- --project ./samples/OptionAnalyzer/OptionAnalyzer.fsproj --analyzers-path ./samples/OptionAnalyzer/bin/Release --verbose
```

Comment:
Would you be game for https://github.com/slaveOftime/Fun.Build?

Reply:
We can give it a try. cc @baronfel

Reply:
@nojaf floated this to me the other day - I'm ok with it, but lacking FSX script caching makes me sad. I'd prefer generally to push data/tasks into MSBuild targets as much as possible because that doesn't require any additional tooling. I understand that not everyone feels the same, however.